devices/families/sdm845-mainline/kernel/default.nix

```diff
@@ -1,28 +1,19 @@
 { mobile-nixos
 , fetchFromGitLab
-, fetchpatch
 , ...
 }:
 
 mobile-nixos.kernel-builder {
-  version = "6.4.0";
+  version = "6.11.0-rc2";
   configfile = ./config.aarch64;
 
   src = fetchFromGitLab {
     owner = "sdm845-mainline";
     repo = "linux";
-    rev = "sdm845-6.4-r1";
-    hash = "sha256-XUYv8tOk0vsG11w8UtBKizlBZ03cbQ2QRGyZEK0ECGU=";
+    rev = "sdm845-6.11.0_rc2-r2";
```

Comment:
There is also a sdm845-6.11.0 tag available, anything blocking on updating to this tag?

Reply:
Now that it's out of RC, we should update before merging indeed..

Reply:
(Thank you @MatthewCroughan for pushing the update)